Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ruby-sys"
-version = "0.2.13"
+version = "0.3.0"
```

Comment:
Please don't include the version change to the PR. I'll bump it myself before releasing.